src/labels.ts

```diff
@@ -0,0 +1,297 @@
+export default {
+  components: {
```

Comment:
Thank you for collocating in a single place. it will help with adding i18n [once it's supported by the plugin API](https://github.com/grafana/grafana/issues/60847)

Reply:
I noticed the `selectors.ts` file contained both `data-test-id` test selectors as well as component labels, so I made this file so that they can be properly separated.

Makes it much easier to rename things 👍